Source/CarthageKit/Xcode.swift

```diff
@@ -962,6 +962,12 @@ private func build(sdk: SDK, with buildArgs: BuildArguments, in workingDirectory
 					let actions: [String] = {
 						var result: [String] = [xcodebuildAction.rawValue]
 
+						if settings.contains(where: { UInt64($0["XCODE_VERSION_ACTUAL"].recover("")) ?? UInt64.zero >= 1200 }) {
+							// Fixes Xcode 12 refusing to link against fat binaries
+							// "Building for iOS Simulator, but the linked and embedded framework 'REDACTED.framework' was built for iOS + iOS Simulator."
+							result += [ "VALIDATE_WORKSPACE=NO" ]
+						}
```

Comment:
We can skip this when using xcframeworks, right?

I'm thinking: pass `options.useXCFrameworks` into `build(sdk:with:in:)` and check it here.